package.json

```diff
@@ -7,9 +7,11 @@
 		"./package.json": "./package.json",
 		".": [
 			{
-				"default": "./index.js"
+				"default": "./index.js",
+				"light": "./light.js"
 			},
-			"./index.js"
+			"./index.js",
+			"./light.js"
```

Comment:
this doesn't do what you think it does; "light" isn't a condition anything checks for, and `./index.js` will always resolve before `./light.js`, so in fact this doesn't expose `light.js` at all.

Reply:
That's an area of package.json where I don't know much about here, so what do you recommand?

Reply:
You'd add, alongside the dot, `"./light": "./light.js"` - but as discussed above, i don't want to add a second export to this package, so it's moot.